lib/diagnostic.js

```diff
@@ -47,6 +47,12 @@ let person2 = {
 
 // Question 3:  Invoke the `fullName()` method so it returns `Santa Clause`?
 let answer3 = null; // replace null with your response
+// not person2.fullName()
+// not fullName(person2);
+// not person1.fullName().person2;
+//not person1.fullName(person2);
+// not person1.fullName.person2;
+// not person1.fullName((person2));
 
```

Comment:
Good job showing what you've tried here!

For this, we want to invoke the fullname() method (defined in person1) but call it on person2.

which looks like `let answer3 =` `person1.fullName.call(person2)`